complete-application/src/components/MenuBar.tsx

```diff
@@ -5,22 +5,11 @@ export default function MenuBar() {
   const {isLoggedIn} = useFusionAuth();
 
   return (
-    <div id="menu-bar" className="menu-bar">
-      {
-        isLoggedIn ? (
-          <>
-            <NavLink to="/make-change" className="menu-link">Make Change</NavLink>
-            <NavLink to="/account" className="menu-link">Account</NavLink>
-          </>
-        ) : (
-          <>
-            <button className="menu-link">About</button>
-            <button className="menu-link">Services</button>
-            <button className="menu-link">Products</button>
-            <button className="menu-link" style={{textDecorationLine: "underline"}}>Home</button>
-          </>
```

Comment:
@BriDavidson was it intended to remove the nav for the signed out state?

Reply:
Yes, those buttons don't redirect anywhere or perform any action so I just removed them since we wanted to get rid of the unused anchors